apple/internal/linking_support.bzl

```diff
@@ -89,7 +85,7 @@ def _register_linking_action(ctx, extra_linkopts = []):
     rpaths = rule_descriptor.rpaths
     linkopts = []
     if rpaths:
-        linkopts.extend(collections.before_each("-rpath", rpaths))
+        linkopts.extend(["-Wl,-rpath,{}".format(rpath) for rpath in rpaths])
```

Comment:
```suggestion
        linkopts.append("-Wl,-rpath,{}".format(rpath) for rpath in rpaths)
```

Reply:
how would this work? don't we need extend since we're adding multiple elements?

Reply:
oh sorry you're right, you still need extend, but at least in py you wouldn't need to form the list normally

Reply:
turns out starlark doesn't support this mode